corehq/apps/app_manager/feature_support.py

```diff
@@ -149,3 +149,10 @@ def supports_update_prompts(self):
         Ability to configure apk/app update checks
         """
         return self._require_minimum_version('2.38')
+
+    @property
+    def enable_remote_requests(self):
+        """
+        Enable Remote Request question type in the form builder.
+        """
+        return self._require_minimum_version('2.40')
```

Comment:
@ctsims the [spec](https://docs.google.com/document/d/1x1xz2N7I_kTUjM4dFx31vt9EuzPrOWzSojnWD062XDs/edit?pli=1#) says "2.40+ (potentially 2.41+)." Do you know which it should be?

Reply:
2.41